### ISSUE TITLE
nether portal doesn't work for illegal connections

### ISSUE THREAD

**Issue by tuxinal** [codehz/ElementZero#152]
hello i am here to report a bug i was trying out your software it works great but when i was trying to enter a nether portal in game it didn't work but my friend who had an xbox account could use the portal (for reasons i can't login to my xbox account on my laptop)

i also tried this on my phone i logged out of my xbox account and i could no longer use the nether portal but with an xbox account i could 

also my character seems to freeze after trying to enter a portal

**Comment by HongyiMC** [comment]
@tuxinal internet connection

**Comment by tuxinal** [comment]
It happens on a self hosted server too I don't know how I can have bad internet connection on a self hosted server

**Comment by madeofstown** [comment]
Try completely removing bedrock edition from your computer and then restarting and then try to reinstall bedrock edition. In order to play online you need a MS account (xbox account) so if you can't sign in on your laptop then you can't play from your laptop. Same thing with your phone. You must have an account of some type if you are able to join the server at all.

**Comment by tuxinal** [comment]
The reason why I am not logged into my xbox is because my country is blocked by the US so xbox can't operate here. If I wanted to I would get a vpn. My server is running in a local network so it doesn't need an xbox account. It's too much of a hassle to use here

**Comment by madeofstown** [comment]
Have you set online mode to disabled in server.properties?

**Comment by tuxinal** [comment]
yes i have, i couldn't join without that. just nether portals don't work

**Comment by madeofstown** [comment]
Ah, have you verified that this bug does not exist in Vanilla BDS on the same world?

**Comment by Weissnix4711** [comment]
> Ah, have you verified that this bug does not exist in Vanilla BDS on the same world?

I agree. Please test in vanilla BDS first. EZ shouldn't really be messing with nether portals anyway, unless you haven't got any custom mods.

Oh, and which version do you use? EZ release, and BDS version.

**Comment by tuxinal** [comment]
I have not tried the windows vanilla BDS as I do not have access to a windows machine at the moment but on linux vanilla BDS it works fine

I have tried two separate servers instantes and neither of them worked


Can someone else test the windows BDS test please?

**Comment by tuxinal** [comment]
i did not find any bugs related to this in the mojang bug report website

**Comment by ub0nFPS** [comment]
It may just be happenstance that the bug occurred while not logged in.

**Comment by tuxinal** [comment]
I have tried this on vanilla BDS and it works fine and have tried it on multiple instances of ez and this problem still occurred

**Comment by elijahr2411** [comment]
Are you still having this issue? If so, try going through a portal on another device so we can see if its a problem with just your game.

**Comment by tuxinal** [comment]
it still doesn't work even on my phone when i log out of my xbox account,
i think this happens because how ez tracks players across dimensions and that it uses xuid for that probably

**Comment by elijahr2411** [comment]
I'll test this when I get the chance.
